sub-groups/sycl-1.2.1/index.md

```diff
@@ -0,0 +1,157 @@
+# Basic Sub group support
+
+This proposal aims to define an interface for using OpenCL 2.2 sub groups in
+SYCL the provisional SYCL 1.2.1 specification, relying on the underlying
+OpenCL implementation supporting the extension `cl_codeplay_basic_subgroups`.
+
+The extension exposes to programmers the ability to identify sub-groups
+on a work-group, count the number of sub-groups available and perform
+a broadcast from one work-item on a sub-group to the rest.
+
+Details of the execution and memory model changes can be found in the
+documentation for the Codeplay's OpenCL vendor extension `cl_codeplay_basic_subgroups` 
+once available.
+
+## Execution model
+
+When this vendor extension is available, the execution model of SYCL 1.2.1
+is extended to also include support for sub-groups of threads inside of a
+work-group.
+Overall, these sub-groups work following the description of the OpenCL 2.2
+sub-groups, with some restrictions:
+
+* The number of sub-groups available for each work-group is determined 
+at compile-time and remains the same during the execution of the SYCL application.
+* The number of threads per sub-group is known at compile-time, and remains the
+same during execution of the SYCL application.
+* Only those functions defined in this proposal are available. 
+In particular, there is no sub-group pipe communication.
+
+## Memory model
+
+Sub-groups can access global and local memory, but, given there is no 
+memory-scope to the atomic or barriers operations in SYCL 1.2.1, there is no
+possibility to specify an equivalent of sub-group memory scope.
+
+## Namespace `basic_sub_group`
+
+All new functionality is exposed under the `basic_sub_group` namespace
```

Comment:
There doesn't seem to be any `basic_sub_group` namespace used in this extension?